.appveyor-build-tag.yml

```diff
@@ -6,6 +6,9 @@ branches:
   except:
     - /nightly_.*/
```

Comment:
The problem is that this will filter branches and not tags. But unfortunately it seems not possible to filter tags: https://help.appveyor.com/discussions/questions/2174-exclude-tags-by-name